### ISSUE TITLE
It was not possible to parse your key, reason: error:0909006C:PEM routines:get_name:no start line

### ISSUE THREAD

**Issue by michelvermeulen** [SocialiteProviders/Providers#639]
Hey folks,

Trying to setup Apple Sign In, I followed the tutorial on https://developer.okta.com/blog/2019/06/04/what-the-heck-is-sign-in-with-apple a gazillion times, re-created client_secret a good 10 times, cleared cache etc... And I'm stuck there.

I do see my key when I check https://appleid.apple.com/auth/keys, and I do have the .env setup properly.

Worth mentioning it worked at some point, but I was getting an invalid_client back from Apple.

Any idea what's wrong here?

**Comment by Adam2Marsh** [comment]
Just to add plus 1 here.

I had this working last year, came to double check it this year and can't get it to work. On the redirect I get the following error:
```
Client error: `POST https://appleid.apple.com/auth/token` resulted in a `400 Bad Request` response: {"error":"invalid_client"}
```

**Comment by Adam2Marsh** [comment]
Okay so I found my issue, it was some bad configuration on my side. I thought I was refreshing my client secret correctly but it was getting overwritten by an old one and I didn't notice. Once I had removed the old config and loaded the new secret it all started to work.

TLDR - Double/Triple check your configuration....

**Comment by atymic** [comment]
Closing due to inactivity
